server/app.js

```diff
@@ -0,0 +1,108 @@
+var express = require('express');
+var index = require('./routes/index');
+
+var debug = require('debug')('absolute-signage:server');
+var http = require('http');
+var app = express();
+
+app.use('/', index);
+
+// catch 404 and forward to error handler
+app.use(function(req, res, next) {
+  var err = new Error('Not Found');
+  err.status = 404;
+  next(err);
+});
+
+// error handler
+app.use(function(err, req, res, next) {
+  // set locals, only providing error in development
+  res.locals.message = err.message;
+  res.locals.error = req.app.get('env') === 'development' ? err : {};
+
+  // render the error page
+  res.status(err.status || 500);
+  res.render('error');
+});
+
+module.exports = app;
+
+/**
+ * Get port from environment and store in Express.
+ */
+var port = normalizePort(process.env.PORT || '3000');
+app.set('port', port);
+
+/**
+ * Create HTTP server.
```

Comment:
Please remove this comment.

It's better to avoid unnecessary comments because it can make the codes unreadable.

For example, the following code gives the impression that you are creating a server, even if you do not comment.

var server = http.createServer(app);